src/calculatePriorWeightsforIHT.jl

```diff
@@ -0,0 +1,135 @@
+"""
+Calculates the Prior Weighting for IHT.
+Returns a weight array (my_snpweights) (1,10000)
+    and a MAF array (my_snpMAF ) (1,10000).
+
+This function updates: hopefully nothing
+"""
+function calculatePriorWeightsforIHT(
+#    xxx        :: SnpData,
+    x        :: SnpLike{2},
+    y        :: Vector{Float64},
+    k        :: Int,
+    v        :: IHTVariable,
+    keyword  :: Dict{AbstractString, Any}
+)
+    # get my_snpMAF from x
+    ALLELE_MAX = 2 * size(x,1)
+    maf, minor_allele, missings_per_snp, missings_per_person = summarize(x)

```

Comment:
`summarize()` is an expensive operation because it requires looping through the whole genotype matrix. Consider making `maf` an input of `calculatePriorWeights`.

Also consider changing the function name to something like `calculate_snp_weights` or something